src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const lowerQuery = query.toLowerCase();
+  
+  const moviesToShow = moviesFromServer.filter(movie => {
+    return movie.title.toLowerCase().includes(lowerQuery)
+  || movie.description.toLowerCase().includes(lowerQuery);
+  });
```

Comment:
still looks complicated :(
DRY 
use destructuring, vars – everything that could make it look simpler

Reply:
repaipred
more symbols, but think looks clearer& simpler)